lib/Widget.js

```diff
@@ -5,7 +5,7 @@ if (!process.browser) {
   require('node-jsx').install({ 'extension': '.jsx' })
 }
 var insomorphic_utils = require('isomorphic-react-boot/lib/utils')
-var Page = require('../jsx/Page.jsx')
+var Main = require('../jsx/Main.jsx')
```

Comment:
Minor nit, could you revert this part of the commit? It was not required to make/change to a Main component. Thanks!